package.json

```diff
@@ -1,28 +1,54 @@
 {
   "name": "single-spa-angular",
-  "version": "3.3.0",
-  "description": "Helpers for building single-spa applications which use Angular 2",
-  "main": "lib/browser-lib/single-spa-angular.js",
-  "schematics": "./schematics.json",
+  "version": "0.0.0",
+  "description": "Helpers for building single-spa applications which use Angular 2+",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/single-spa/single-spa-angular.git"
+  },
+  "keywords": [
+    "single-spa",
+    "angular"
+  ],
+  "author": "Joel Denning",
+  "maintainers": [
+    "Joel Denning",
+    "Artur Androsovych"
+  ],
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/single-spa/single-spa-angular/issues"
+  },
+  "homepage": "https://github.com/single-spa/single-spa-angular#readme",
+  "private": true,
   "scripts": {
     "prepublishOnly": "yarn build && yarn test",
-    "build": "yarn clean && yarn build:webpack && yarn parcel && yarn spa && yarn :copy-schematic-files",
-    "clean": "rimraf lib parcel",
-    "test": "jest",
+    "clean": "rimraf lib pacel",
+    "build": "run-s clean build:single-spa-angular build:webpack build:schematics",
+    "test": "yarn clean && yarn jest",
     "lint": "eslint ./src/**/*.ts",
     ":copy-schematic-files": "cpx \"src/**/_files/**/**\" lib",
     "parcel": "ngc -p tsconfig.parcel.json",
-    "spa": "tsc -p tsconfig.spa.json",
-    "build:webpack": "tsc -p tsconfig.node.json",
+    "// - SINGLE-SPA-ANGULAR": "Scripts for building single-spa-angular",
+    "build:single-spa-angular": "yarn ts-node tools/build",
+    "// - WEBPACK": "Scripts for building Webpack config transformer",
+    "prebuild:webpack": "rimraf lib/lib",
```

Comment:
Why is there a lib/lib directory?

Reply:
webpack is emitted there

Reply:
Why then is the import for webpack at `single-spa-angular/lib/webpack` instead of `single-spa-angular/lib/lib/webpack`?

https://github.com/single-spa/single-spa-angular/pull/167/files#diff-def93996c46300cb52885ed7c3584275R1

Reply:
This question is the main one I'm waiting on before reviewing again to approve/merge ^

Reply:
The content of the lib folder is copied into node_modules/single-spa-angular folder